CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## 0.9.0 (2022-06-07)
+
+### -Added new option `authorizationClient`
+
+Defines Authorization client to use to get access token to Context Share API (authority: <https://ims.bentley.com> )
```

Comment:
Override the default access token used by the `RealityDataClient` with an access token from your `authorizationClient`